changelog.yaml

```diff
@@ -18,32 +18,43 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:

```

Comment:
Don't we need to provide `version` and `date` fields?

Reply:
Not in the changelog.yaml. And in Chart.yaml it has already been updated - patch digit is ok from my perspective.